_source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: blog_post
+title: "Developer Day 2021:  Authentication, Authorization, and Celebrating Our Diverse Developer Community"
```

Comment:
```suggestion
title: "Developer Day 2021:  Authentication, Authorization, and Celebrating Our Diverse Developer Community"
```
title: "Developer Day 2021: Celebrating the Future of Identity"

Updating to the current title